### PR TITLE
feat : Adding Button Component to library

### PR DESCRIPTION
**Pull request by Bharatrathod-81**
Added various types of button component and  made it responsiveness

### CODE REVIEW

File: Components/buttons/Button.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
+.primary,.Link{
```
Comment (junaidshaikh-js) [inline Components/buttons/Button.css]: each selector should be on new line

File: Components/buttons/Button.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    padding: 1.5rem 3rem;
+    margin: 1.5rem;
+    background-color: rgb(24, 130, 252);
+    color: white;
+    font-size: larger;
+    font-weight: bold;
+    cursor: pointer;
+}
+.cornerRadiousBtn{
```
Comment (junaidshaikh-js) [inline Components/buttons/Button.css:12]: keep a line blank between two blocks

File: Components/buttons/Button.html [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    <link rel="stylesheet" href="">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <nav class="nav-body">
+        <ul class="list-col-left">
+            <li><a class="atribute" href="/">Home</a></li>
+            <li><a class="atribute" href="./Components/Avatar/Avatar.html">Documentation</a></li>
+            <ul class="list-col-right">
```
Comment (junaidshaikh-js) [inline Components/buttons/Button.html:20]: wrap this ul inside li as `li` should be the direct child of `ul`

File: Components/buttons/Button.html [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+                <li class="componentlist"><a href="/Components/Badge/Badge.html">Badge</a></li>
+                <li class="componentlist"><a href="/Components/Alert/Alert.html">Alert</a></li>
+                <li class="componentlist"><a href="./Button.html">Button</a></li>
+                <li class="componentlist"><a href="">Card</a></li>
+                <li class="componentlist"><a href="">Image</a></li>
+                <li class="componentlist"><a href="">Input</a></li>
+            </ul>
+        </div>
+        <div class="discriptionBar">
+            <h1 class="margin2rem">Button</h1>
```
Comment (junaidshaikh-js) [inline Components/buttons/Button.html]: There should be only one `h1` per page

**Review by junaidshaikh-js — COMMENTED** [review]
Good Work!

File: Components/buttons/Button.html (hunk 4 above)
Comment (Bharatrathod-81) [reply Components/buttons/Button.html]: thank you ..i didn't notice such thing but i'll resolve it

**Review by nikhilmalviya-dev — APPROVED** [review]
Great work

**Pull request merged**